pkgs/development/python-modules/secretstorage/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, pkgs, buildPythonPackage, dbus-python, python, cryptography }:
+
+buildPythonPackage rec {
+  pname = "secretstorage";
+  version = "2.3.1";
+  name = "${pname}-${version}";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "mitya57";
+    repo = "secretstorage";
+    rev = "2636a47b45aff51a21dfaf1cbd9f1f3c1347f7f4";
```

Comment:
This should just be `"${version}"`. And note the sha256 will therefore also change.